### PR TITLE
Fix Double Chests

### PR DESCRIPTION
**Pull request by FlamingKnight**
Just remaking this PR as it has the commits only related to fixing the comparators, not anything else, and to fix the random conflicting files for languages

### CODE REVIEW

**Review by AlicanCopur — COMMENTED** [review]
Tested and works well.

**Comment by joserobjr** [comment]
Thank you for the PR and for taking the time to attempt to improve PowerNukkit. But unfortunately, after reviewing, I don't think this is the right way to solve the issue #1174 

The proposed change is removing the part of the setContents methods that actually set the contents in the inventory.

I'm going to work on this issue now and I will submit a new PR when the new solution is completed.

File: src/main/java/cn/nukkit/inventory/DoubleChestInventory.java [hunk 1]
```diff
@@ -102,8 +102,6 @@ public void setContents(Map<Integer, Item> items) {
                 } else if (this.right.slots.containsKey(i - this.left.size)) {
                     this.clear(i);
                 }
-            } else if (!this.setItem(i, items.get(i))) {
-                this.clear(i);
```
Comment (joserobjr) [inline src/main/java/cn/nukkit/inventory/DoubleChestInventory.java:106]: Sorry, these lines can't be deleted

**Review by joserobjr — CHANGES_REQUESTED** [review]

File: src/main/java/cn/nukkit/inventory/DoubleChestInventory.java (hunk 1 above)
Comment (roridev) [reply src/main/java/cn/nukkit/inventory/DoubleChestInventory.java:106]: yeah that was really fishy. since i'm not used to that api i couldn't review it.

**Comment by FlamingKnight** [comment]
Ok thanks, from what I checked it wasn't affecting anything but I bet there is a better solution!

**Pull request closed without merging**